developer-experience/recipes/execute-local-pipeline.md

```diff
@@ -0,0 +1,32 @@
+# Execute Local Pipeline
+
+## Abstract
+
+Having the ability to execute pipeline activities locally has been identified as an opportunity to promote positive developer experience.
+In this document we will explore a solution which will allow us to have the local CI experience to be as similar as possible to the remote process in the CI server.
+
+Using the suggested method will allow us to:
+
+1. Build
+2. Lint
+3. Unit test
+4. E2E test
+5. Deploy
```

Comment:
nit: i would suggest to reword this to maybe Run Solution or something like that. In the rest of the content, i got the impression this was more geared towards how we could get services running so we run tests, etc. This doesnt seem to require us to "deploy" a container to AKS, ACI, or similar (which is actually a good thing!).